playground/mocks/idp/idx/data/authenticator-select-enroll-options.json

```diff
@@ -82,14 +81,20 @@
     "type": "array",
     "value": [
       {
-        "authenticatorType": "password",
-        "authenticatorId": "autwa6eD9o02iBbtv0g3",
-        "authenticatorName": "Okta Password"
+        "label": "aaaaa",
+        "value": {
+          "type": "password",
+          "id": "autwa6eD9o02iBbtv0g3",
+          "name": "Okta Password"
```

Comment:
I guess the name is what goes into the label right 

https://oktawiki.atlassian.net/wiki/spaces/eng/pages/1031032455/Embedded+Objects+in+IdX+Authenticators+API#Proposed-End-State

Reply:
I'm not sure if there will be name eventually. This is refactoring from current API structure.
Widget display `label` in the remediation option.